### DISCUSSION TITLE
[Portal Engine] Public share error

### DISCUSSION THREAD

**Discussion by Maksonis** [pimcore/pimcore#12173] [Q&A / Support]
When I try to do a public share of assets I get the following error:

- Locally:
![image](https://user-images.githubusercontent.com/17277953/169492782-b5edda17-e2a9-4af2-bd2d-7201f06eee9a.png)

- Live demo
![image](https://user-images.githubusercontent.com/17277953/169492891-904aa933-7ebc-4862-93ce-7a92cc438c53.png)

Is that a bug ?

**Comment by fashxp** [comment]
hmm, cannot reproduce that on the public demo. 
can you provide some more information?

**Reply by Maksonis** [reply]
@fashxp thanks for your reply

I investigated little bit more and find out that error is here if checkbox Enable Language Redirect is not checked.

![image](https://user-images.githubusercontent.com/48808653/170949393-b9c7bb39-a929-457b-9aa3-c5025e780144.png)

Since languageVariants array is empty, error is triggered.

**Reply by fashxp** [reply]
strange ... do you also have a stacktrace at hand?

**Reply by Maksonis** [reply]
here it is
![image](https://user-images.githubusercontent.com/17277953/170982566-55bac196-a90a-49ed-a8a2-612f2f680229.png)

**Reply by fashxp** [reply]
still cannot reproduce it. but added an additional check in lastest bugfix release (2.2.6) that should do the trick.

**Reply by Maksonis** [reply]
Thanks a lot!
